server/odc-core/src/main/java/com/oceanbase/odc/core/sql/split/SqlCommentProcessor.java

```diff
@@ -599,4 +609,71 @@ public int getValue() {
         }
     }
 
+    public class SqlIterator implements Iterator<String>, AutoCloseable {
```

Comment:
don't do this, make `SqlCommentProcessor` as an util.

the internal status of `SqlCommentProcessor` is much too complex that should not care about it.

from my point of view, `addLineMySQL` and `addLineOracle` is enough

Reply:
done. Used a static method `getSqlLines` to return inner static class `SqlLine`. The object of `SqlLine` will create a new `SqlCommentProcessor` for usage.